### ISSUE TITLE
M3U file not importing (with proof) 

### ISSUE THREAD

**Issue by Lyeco** [OpenEmu/OpenEmu#3048]
I followed the instructions carefully and I am very stuck at the last part. Apparently, the m3u file didn't import and I can't play my PS1 game with multiple disc.

Web proof : http://i.imgur.com/0MssuoY.gifv

![ezgif com-0142605856](https://cloud.githubusercontent.com/assets/15318314/22853743/21e99a38-f099-11e6-8d1e-dfd3cde2116f.gif)

**Comment by clobber** [comment]
https://github.com/OpenEmu/OpenEmu/issues/2992#issuecomment-270859623

**Comment by Lyeco** [comment]
How do you make the m3u "be saved in a folder together with all the cue sheets and their associated files"?

**Comment by clobber** [comment]
From [the guide](https://github.com/OpenEmu/OpenEmu/wiki/User-guide:-CD-based-games#q-i-have-a-multi-disc-game):

* You need to create a **plain text** .m3u file containing the filenames of all the cue sheets for the game.
* The file **must be saved in a folder together with all the cue sheets and their associated files** in order for them to be automatically imported.
* Also, the cue sheets for the game must not already be pre-imported in your library.

So,

```
Folder containing all Final Fantasy VII files/
└╴Final Fantasy VII (Disc 1).bin
└╴Final Fantasy VII (Disc 1).cue
└╴Final Fantasy VII (Disc 2).bin
└╴Final Fantasy VII (Disc 2).cue
└╴Final Fantasy VII (Disc 3).bin
└╴Final Fantasy VII (Disc 3).cue
└╴Final Fantasy VII.m3u    <-- import this file
```
Sorry, I don't know how to explain it more simply than that. Our guide is straight forward and step-by-step.

**Comment by Clayton-Sodergren** [comment]
I'm still having an issue, I think my problem is that I imported the .cue files individually before learning how you're supposed to do it. How can I find and delete these .cue files, since the game isn't showing up?

**Comment by clobber** [comment]
@Clayton-Sodergren 

* Remove whatever you first imported from your Library.
* Check the "All Games" collection to see all imported games.
* Make sure you did not manually copy files to Game Library folder (please don't try to outsmart the importer).
* Read both of our disc guides ([User guide: CD based games](https://github.com/OpenEmu/OpenEmu/wiki/User-guide:-CD-based-games#q-i-have-a-multi-disc-game) & [User Guide: Required m3u file missing](https://github.com/OpenEmu/OpenEmu/wiki/User-Guide:-Required-m3u-file-missing)).

If an M3U file doesn't importer, it's because you don't have something right so follow the guides closely.
